changelogs/fix_tostring.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
- Could you please have fragment with indicator as `bugfixes` and move this file to  `changelogs/fragments` directory, example: https://github.com/ansible-collections/junipernetworks.junos/pull/293/files